task-manager.dart

```diff
@@ -45,14 +45,27 @@ class Task {
 }
 
 class TaskManager {
+
+  List<Task> _tasks = [];
+
   // Add task method
   void addTask() {}
 
   // View all tasks
   void viewAllTask() {}
 
   // View all completed tasks
-  void viewCompletedTasks() {}
+  void viewCompletedTasks() {
+    List<Task> completedTasks = _tasks.where((task) => task.getStatus() == Status.completed).toList();
+
+    if (completedTasks.isEmpty) {

```

Comment:
nit consider adding a blank line before the if statement for better readability.